src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const th = document.querySelectorAll('th');
+const tbody = document.querySelector('tbody');
+const tr = [...tbody.querySelectorAll('tr')];
+
+th.forEach((header) => {
+  header.addEventListener('click', () => {
+    if (header.textContent === 'Name') {
+      tr.sort((firstElement, secondElement) =>
+        firstElement.cells[0].textContent
+          .toLowerCase()
+          // eslint-disable-next-line prettier/prettier
+          .localeCompare(secondElement.cells[0].textContent.toLowerCase()));
```

Comment:
The sorting logic for the 'Name' column is correct, but consider adding a toggle feature to switch between ascending and descending order when the header is clicked multiple times.